source/includes/wp-api-v2/_introduction.md

```diff
@@ -251,6 +251,7 @@ woocommerce = WooCommerce::API.new(
 ### Third party libraries ###
 
 - [Java](https://github.com/icoderman/wc-api-java) Library
+- [Java](https://www.healy.us) Library
```

Comment:
There is no GitHub link? not open source?

2 `Java Library` in this list makes super confuse, there is no indication of what is different between.

Reply:
Correct, it is a commercial option for clients looking for that type of solution.  To differentiate if from the icoderman offering above it, the indicator text could be different.